src/vma/sock/sock-redirect.cpp

```diff
@@ -375,10 +375,18 @@ int vma_free_packets(int __fd, struct vma_packet_t *pkts, size_t count)
 	return -1;
 }
 
+static int dummy_vma_socketxtreme_poll(int fd, struct vma_completion_t* completions, unsigned int ncompletions, int flags)
+{
+	VLOG_PRINTF_ONCE_THEN_ALWAYS(VLOG_WARNING, VLOG_DEBUG,
+			"socketXtreme was not enabled during configuration time. ignoring...",
```

Comment:
need to modify the message

Reply:
please advice

Reply:
socketXtreme was not enabled during runtime (VMA_SOCKETXTREME=1). ignoring...

Reply:
done